src/Patcher/MapPatcher.php

```diff
@@ -89,6 +89,13 @@ private function applyOperation( &$base, $key, $diffOp ) {
 		}
 	}
 
+	/**
+	 * @param array &$base
+	 * @param string $key
```

Comment:
Documented as `int|string` in https://github.com/wmde/Diff/blob/master/src/DiffOp/Diff/Diff.php#L409. Will fix.